src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -215,4 +215,10 @@ public void matchUserToPerson(String name) {
             }
         }
     }
+
+    @Override
+    public User getUser() {
+        //TODO Can you do this? Must you create a new object to be returned instead?
```

Comment:
@nianfei97 Need your opinion on this

Reply:
In this case, returning the user directly should be OK. The reason for copy gets (create new object and return that) is to prevent other people calling your methods from modifying your original object. But in this case, since user is immutable, it doesn't really matter all that much.